src/main/java/org/flickit/flickitassessmentcore/application/service/assessment/CreateAssessmentService.java

```diff
@@ -24,6 +24,8 @@
 @Transactional
 public class CreateAssessmentService implements CreateAssessmentUseCase {
 
+    public static final long NOT_DELETED_DELETION_TIME = 0L;
```

Comment:
چون تو سرویس GetAssessmentList هم از این constant داره استفاده می‌شه و خاص create نیست، به نظر میاد بهتره منتقلش کنیم به یه جای دیگه‌ای.
من این به ذهنم می‌رسه:
تو هم لایه سرویس، یه پکیج constant بسازیم.
داخل اون پکیج یک کلاس `AssessmentConstants` رو به صورت زیر تعریف کنیم:

```java
public final class AssessmentConstants {
    public static final long NOT_DELETED_DELETION_TIME = 0L;

    private AssessmentConstants() {
        // Private constructor to prevent instantiation
    }
}
```

اگر پیشنهاد بهتری به ذهنتون می‌رسه بفرمایید.

Reply:
منم موافقم.